### PR TITLE
Create 2023.05.30_meeting_notes.md

### LINKED ISSUE #234: OpenJS Foundation Standards Working Group Meeting 2023-05-30
## Time

**UTC Tue 30-May-2023 17:00 (05:00 PM)**:

| Timezone      | Date/Time             |
|---------------|-----------------------|
| US / Pacific  | Tue 30-May-2023 10:00 (10:00 AM) |
| US / Mountain | Tue 30-May-2023 11:00 (11:00 AM) |
| US / Central  | Tue 30-May-2023 12:00 (12:00 PM) |
| US / Eastern  | Tue 30-May-2023 13:00 (01:00 PM) |
| EU / Western  | Tue 30-May-2023 18:00 (06:00 PM) |
| EU / Central  | Tue 30-May-2023 19:00 (07:00 PM) |
| EU / Eastern  | Tue 30-May-2023 20:00 (08:00 PM) |
| Moscow        | Tue 30-May-2023 20:00 (08:00 PM) |
| Chennai       | Tue 30-May-2023 22:30 (10:30 PM) |
| Hangzhou      | Wed 31-May-2023 01:00 (01:00 AM) |
| Tokyo         | Wed 31-May-2023 02:00 (02:00 AM) |
| Sydney        | Wed 31-May-2023 03:00 (03:00 AM) |

Or in your local time:
* https://www.timeanddate.com/worldclock/fixedtime.html?msg=Node.js+Foundation+Standards%20Working%20Group+Meeting+2023-05-30&iso=20230530T1700
* or https://www.wolframalpha.com/input/?i=05PM+UTC%2C+May+30%2C+2023+in+local+time

## Links

* Minutes Google Doc: <https://docs.google.com/document/d/1-eLDNvB0oWuH2e3-jbyLtzKXS-0dBsWU6zZLc0UGass/edit>

## Agenda

Extracted from **standards-agenda** labelled issues and pull requests from the **openjs-foundation org** prior to the meeting.

### openjs-foundation/standards

* Discuss `package.json` standardization and governance [#233](https://github.com/openjs-foundation/standards/issues/233)
* W3C AB Election Ballot Due June 1 [#232](https://github.com/openjs-foundation/standards/issues/232)
* W3C TPAC 11-15 September 2023 - Break out session? [#231](https://github.com/openjs-foundation/standards/issues/231)
* Add Tobie Langel as OSI Representative [#224](https://github.com/openjs-foundation/standards/pull/224)
* Define terms for SME Participation [#173](https://github.com/openjs-foundation/standards/issues/173)
* Stepping down as OSI Affiliate Representative [#170](https://github.com/openjs-foundation/standards/issues/170)

## Invited

* OpenJS Foundation Cross Project Council
* OpenJS Foundation Project Maintainers
* OpenJS Foundation Board of Directors

## Notes

The agenda comes from issues labelled with `standards-agenda` across **all of the repositories in the openjs-foundation org**. Please label any additional issues that should be on the agenda before the meeting starts.

## Joining the meeting

link for participants: Zoom link: <https://zoom.us/j/599005386>

* For those who just want to watch: <https://livestream.openjsf.org>

---

**Invitees**

Please use the following emoji reactions in this post to indicate your
availability.

* :+1: - Attending
* :-1: - Not attending
* :confused: - Not sure yet

### PR DESCRIPTION
**Pull request by jorydotcom**
Add Meeting Notes - closes #234

### CODE REVIEW

**Review by ljharb — APPROVED** [review]

**Review by joesepi — APPROVED** [review]

File: meeting-notes/2023.05.30_meeting_notes.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+- Jory Burson (@jorydotcom)      
+- Lea Verou (@leaverou) 
+- Leo Cunha
+- Jordan Harband (@ljharb) 
+- Joe Sepi (@joesepi)
+- Robin Ginn (@rginn) 
+- Tobie Langel (@tobie) 
+- Michael Champion (@michaelchampion)
+- Darcy Clarke (@darcyclarke) 
+- Sendil Kumar (@sendilkumarn)
```
Comment (VoltrexKeyva) [inline meeting-notes/2023.05.30_meeting_notes.md:20]: Were Leo Cunha and Sendil Kumar actually present in the meeting? Looking at the recording I didn't see them unless I'm missing something, did they join after the recording ended?

**Review by VoltrexKeyva — APPROVED** [review]

File: meeting-notes/2023.05.30_meeting_notes.md (hunk 1 above)
Comment (VoltrexKeyva) [reply meeting-notes/2023.05.30_meeting_notes.md:20]: (Unrelated but I just noticed that I'm not a member of the standards WG even though I'm an active member in the README, is that intentional?)

Comment (ljharb) [reply meeting-notes/2023.05.30_meeting_notes.md:20]: Thanks, I just updated the team to match the readme.

Comment (ljharb) [reply meeting-notes/2023.05.30_meeting_notes.md:20]: Leo was certainly present as he was explicitly welcomed; I assume that Sendil was as well altho I don't think they spoke?

Comment (VoltrexKeyva) [reply meeting-notes/2023.05.30_meeting_notes.md:20]: Ah I see, though yeah not sure about Sendil, maybe @jorydotcom can give an insight?

**Review by DerekNonGeneric — APPROVED** [review]

File: meeting-notes/2023.05.30_meeting_notes.md (hunk 1 above)
Comment (jorydotcom) [reply meeting-notes/2023.05.30_meeting_notes.md:20]: Sendil joined late but did not speak. By default YouTube won't show all the participants in the recording - just the ones who spoke.

Comment (VoltrexKeyva) [reply meeting-notes/2023.05.30_meeting_notes.md:20]: Ah thank you for the insight!

**Pull request merged**